core/src/main/java/io/parsingdata/metal/Shorthand.java

```diff
@@ -90,10 +91,10 @@ private Shorthand() {}
     public static Token def(final String name, final long size, final Encoding encoding) { return def(name, con(size), encoding); }
     public static Token def(final String name, final long size) { return def(name, size, (Encoding)null); }
     public static final Token empty = def(NO_NAME, 0L);
-    public static Token cho(final String name, final Encoding encoding, final Token... tokens) { return new Cho(name, encoding, tokens); }
-    public static Token cho(final String name, final Token... tokens) { return cho(name, null, tokens); }
-    public static Token cho(final Encoding encoding, final Token... tokens) { return cho(NO_NAME, encoding, tokens); }
-    public static Token cho(final Token... tokens) { return cho((Encoding)null, tokens); }
+    public static Token cho(final String name, final Encoding encoding, final Token token1, final Token token2, final Token... tokens) { return new Cho(name, encoding, concatVarArgTokens(token1, token2, tokens)); }
```

Comment:
Agreed! There doesn't seem to be a valid reason to have them be different... Plus it requires code in the `Shorthand`, which we try to avoid.